src/main/java/roomescape/repository/ReservationTimeRepository.java

```diff
@@ -0,0 +1,28 @@
+package roomescape.repository;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.ReservationTime;
+
+@Repository
+public interface ReservationTimeRepository extends JpaRepository<ReservationTime, Long> {
+
+    Optional<ReservationTime> findByStartAt(LocalTime startAt);
+
+    @Query("""
+            SELECT rt 
```

Comment:
```suggestion
            SELECT rt.id
```

로직상 id만 받아되 되지 않나 싶네요!

Reply:
`ReservationTimeApiController`의 `getReservationTimesIsBooked()`에서 response를 보낼 때 `startAt`을 같이 보내서 객체 전체의 값이 필요할 것 같습니다!